### PR TITLE
[BIOMAGE-1682] - Add comparison type to DE body

### PR DESCRIPTION
**Pull request by aerlaut**
# Description
Add `comparisonType` to the API specification.

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-1682

#### Link to staging deployment URL (or set N/A)
https://ui-agi-ui622-api295.scp-staging.biomage.net/

#### Links to any PRs or resources related to this PR
https://github.com/hms-dbmi-cellenics/ui/pull/622

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [X] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [X] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [X] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [X] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [x] Started end-to-end tests on the latest commit.

### Documentation updates
- [X] Relevant Github READMEs updated **or** no GitHub README updates required.
- [X] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: .github/workflows/pr_validate.yaml [hunk 1]
```diff
@@ -204,11 +204,6 @@ jobs:
 
           echo "::set-output name=body::$UNCHECKED_BODY"
 
-      - id: check-case-sensitivity
```
Comment (cosa65) [inline .github/workflows/pr_validate.yaml:207]: Why was this removed?

**Review by cosa65 — APPROVED** [review]
One small comment

File: .github/workflows/pr_validate.yaml (hunk 1 above)
Comment (aerlaut) [reply .github/workflows/pr_validate.yaml:207]: This is a left over debug step leftover from the PR which fixed the uppercase branch name (#291).
